gatsby-node.js

```diff
@@ -1,22 +1,20 @@
 const path = require("path")
 
+const localPlugins = [
+  "contentful-plugin",
+  "datocms-plugin",
+  "drupal-plugin",
+  "wordpress-plugin",
+]
+
 // for theming development
 exports.onCreateWebpackConfig = ({ actions, store }) => {
   // redux state to check which plugin is used
   const state = store.getState()
   const plugins = state.config.plugins.map((plugin) => plugin.resolve)
   const plugin = plugins.reduce((a, b) => {
     if (a) return a
-    switch (b) {
-      case "gatsby-source-contentful":
```

Comment:
Because themes can be mixed and matched with different CMSs, this would no longer reliably work for the dev repo, so it checks for the local plugin name instead